metadata/targets.json

```diff
@@ -133,18 +133,18 @@
    },
    "signing_config.v0.2.json": {
     "hashes": {
-     "sha256": "149b5d01e0cc43314508c2938dcfe6bad3148f0a5b0837944c33822be9143448"
+     "sha256": "c1e5ffb69679205969c7f731eb6d39ecd4ee2daeaccde216db61aa738a8fd347"
     },
-    "length": 1223
+    "length": 1463
    },
    "trusted_root.json": {
     "hashes": {
-     "sha256": "ed6a9cf4e7c2e3297a4b5974fce0d17132f03c63512029d7aa3a402b43acab49"
+     "sha256": "d186ec66ca32275d48a54f0d741d08412ecfad3412561a77ebd1dadff6a3e8fe"
     },
-    "length": 6824
+    "length": 7287
    }
   },
-  "version": 18,
```

Comment:
@jku I updated this PR with another commit and it bumped the version again. Is this a bug in tuf-on-ci, where the version will be bumped not based on what's at HEAD but what's in the PR?

For now, I'll just open a new PR.